Tasks/merge.js

```diff
@@ -1,19 +1,32 @@
-// Merge two dictionaries
+'use strict';
+//Merge two dictionaries
 
-merge_two_objects=(function merge_two_objects(object_1,object_2){
-    object_3 = [({})][0];
-   for (attribute_name in object_1) {
-       (
-          object_3[attribute_name] == object_1[attribute_name]
-       );
-        object_3[attribute_name] = object_1[attribute_name]
-  }
-   for (attribute_name in object_2) {
-      (object_3[attribute_name] == object_2[attribute_name])
-     object_3[attribute_name] = object_2[attribute_name]
-    }
-return object_1, object_2, object_3;
-})
+const mergeTwoObjects = function(objFirst, objSecond) {
+  let objFinal = {};
 
-const result = merge_two_objects({ a: 'uno', b: 'due' }, { c: 'tre' });
+  for (const key in objFirst) {
```

Comment:
Prefer for..of

Reply:
(for (const key of Object.keys(objFirst)) {)- like this ?